executor/new_index_lookup_join.go

```diff
@@ -148,12 +155,19 @@ func (e *NewIndexLookUpJoin) newOuterWorker(resultCh, innerCh chan *lookUpJoinTa
 }
 
 func (e *NewIndexLookUpJoin) newInnerWorker(taskCh chan *lookUpJoinTask) *innerWorker {
+	// Since multiple inner workers runs at one time, we copy the ranges.
```

Comment:
how about: `Since multiple inner workers running concurrently, we should copy "e.indexRanges" for every worker to avoid data race.`